docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+# Inspired by https://docs.docker.com/compose/django/
+version: '3.4'
+
+services:
+  db:
+    image: mdillon/postgis
+    # environment:
+    #   - POSTGRES_USER=${PG_USER:zrc}
+    #   - POSTGRES_PASSWORD=${PG_PASSWORD:zrc}
+  web:
+    image: stevenbal/open-zaak
```

Comment:
@sergei-maertens What's best practice here?

`build: .` or an image of self like steven did?

In any case, we now have: https://hub.docker.com/r/openzaak/open-zaak You can push it there.